packages/cf-core/src/cf-base.less

```diff
@@ -483,6 +483,19 @@ tbody tr {
 th {
     font-weight: 600;
     text-align: left;
+
```

Comment:
Let's add a code comment here. Superficially it seems like the real issue is that heading elements shouldn't be placed in table headings, however, they are needed for accessibility semantics as I understand? So maybe say:

```
// Heading elements may at times appear inside th elements as required for navigating the page's content with screenreaders. 
```

It may also be worth adding a note in the cf-tables usage doc for this as well.

Reply:
Good call.

Reply:
@anselmbradford Updated. LMK what you think.